packages/logger/src/logger-event-measurements.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// tslint:disable-next-line: no-empty-interface
+export interface BaseTelemetryMeasurements {
+    [name: string]: number;
+}
+
+export interface BatchScanRequestMeasurements extends BaseTelemetryMeasurements {
+    totalScanRequests: number;
+    acceptedScanRequests: number;
+    rejectedScanRequests: number;
+}
+
+export interface BatchPoolMeasurements extends BaseTelemetryMeasurements {
+    runningTasks: number;
+    samplingIntervalInSeconds: number;
+    maxParallelTasksPossible: number;
```

Comment:
this is max actual running task, right?
should be just maxParallelTasks?

Reply:
fixed